src/robot/libraries/Collections.py

```diff
@@ -220,13 +242,17 @@ def get_slice_from_list(self, list_, start=0, end=None):
         | ${z} = ['a', 'b', 'c']
         | ${L5} is not changed
         """
-        self._validate_list(list_)
-        start = self._index_to_int(start, True)
-        if end is not None:
-            end = self._index_to_int(end)
+        if not start:
+            start = 0
         return list_[start:end]
 
-    def count_values_in_list(self, list_, value, start=0, end=None):
+    def count_values_in_list(
+        self,
+        list_: list,
```

Comment:
Type much be `Sequence`. Custom sequences may handle `count` differently than `list` and we want to honor them.